FrenchFilter/sections/specific.txt

```diff
@@ -2,6 +2,7 @@
 !-------------------- Specific rules --------------------!
 !--------------------------------------------------------!
 !
+01net.com##.mrf-adv__wrapper
```

Comment:
I can't find it on the desktop with French IP, could you move the rule to the Mobile filter - specific_web.txt?

Reply:
https://github.com/AdguardTeam/AdguardFilters/blob/f21aed9a35bd7d73a53d003886ded77666ff2e9f/MobileFilter/sections/specific_web.txt#L295
I see that element without mobile filter only.
@ryanbr Which filter URL is used in Brave?

Reply:
we're strictly using the French filterset here